src/functionalTest/resources/features/F-206 - Request Notice Of Change/common/verify-case-events/F-206_Verify_NoC_Request_Event_Data_Alice.td.json

```diff
@@ -0,0 +1,43 @@
+{
+  "_guid_": "F-206_Verify_NoC_Request_Event_Data_Alice",
+  "_extends_": "F-206_Verify_NoC_Request_Event_Data__Base",
+
+  "specs": [
+    "to get Case Events API returns a NoCRequest event in which the user ID is set to Alice's email address AND the proxied_by field set to the ID of the system user"
+  ],
+
+  "expectedResponse": {
+    "_extends_": "Common_200_Response",
+    "body" : {
+      "auditEvents": [
+        {
+          "id": "[[ANYTHING_PRESENT]]",
+          "description" : "[[ANYTHING_PRESENT]]",
+          "user_id": "${[scenarioContext][parentContext][testData][users][invokingUser][id]}",
+          "user_last_name": "Solicington",
+          "user_first_name": "Alice",
+          "event_name": "[[ANYTHING_PRESENT]]",
+          "created_date": "[[ANYTHING_PRESENT]]",
+          "case_type_id": "[[ANYTHING_PRESENT]]",
+          "case_type_version": "[[ANYTHING_PRESENT]]",
+          "state_id": "[[ANYTHING_PRESENT]]",
+          "state_name": "[[ANYTHING_PRESENT]]",
+          "data": "[[ANYTHING_PRESENT]]",
+          "data_classification": "[[ANYTHING_PRESENT]]",
+          "security_classification": "[[ANYTHING_PRESENT]]",
+          "significant_item": "[[ANY_NULLABLE]]",
+          "security_classifications": "[[ANYTHING_PRESENT]]",
+          "summary": "[[ANY_NULLABLE]]",
+          "proxied_by": "[[ANYTHING_PRESENT]]",
```

Comment:
This doesn't strictly do what the spec says, although I'm not sure how you'd get the ID of the system user from IDAM, to compare against the actual returned value?